docs/content/contributing/submitting-pull-requests.md

```diff
@@ -3,12 +3,214 @@ title: "Traefik Pull Requests Documentation"
 description: "Looking to contribute to Traefik Proxy? This guide will show you the guidelines for submitting a PR in our contributors guide repository."
 ---
 
-# Submitting Pull Requests
+# Before You Submit a Pull Request
+
+This guide is for contributors who already have a pull request to submit. 
+If you're looking for information on setting up your developer environment 
+and creating code to contribute to Traefik Proxy or related projects, 
+see the [development guide](https://docs.traefik.io/contributing/building-testing/).
+
+Looking for a way to contribute to Traefik Proxy? 
+Check out this list of [Priority Issues](https://github.com/traefik/traefik/labels/contributor%2Fwanted), 
+the [Good First Issue](https://github.com/traefik/traefik/labels/contributor%2Fgood-first-issue) list, 
+or the list of [confirmed bugs](https://github.com/traefik/traefik/labels/kind%2Fbug%2Fconfirmed) waiting to be remedied.
+
+# How We Prioritize
+We wish we could review every pull request right away. 
+Unfortunately, our team has to prioritize pull requests (PRs) for review 
+(but we are welcoming new [maintainers](https://github.com/traefik/traefik/blob/master/docs/content/contributing/maintainers-guidelines.md) to speed this up, so if you are interested, check it out and apply). 
+
+The PRs we are able to handle fastest are:
+* Documentation updates
+* Bug fixes
+* Enhancements and Features with a contributor/wanted tag
+
+PRs that take more time to address include:
+* Enhancements or Features without the contributor/wanted tag  
+  
+If you have an idea for an enhancement or feature that you would like to build, 
+[create an issue](https://github.com/traefik/traefik/issues/new/choose) for it first 
+and tell us you are interested in writing the PR. 
+If an issue already exists, definitely comment on it to tell us you are interested in creating a PR. 
+
+This will allow us to communicate directly and let you know if it is something we would accept. 
+It also allows us to make sure you have all the information you need during the design phase 
+so that it can be reviewed and merged quickly. 
+  
+If you have questions about the Triage process, [read more here](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).  
+  
+# The Pull Request Submit Process
+
+Merging a PR requires the following steps to be completed before the pull request is merged automatically.
```

Comment:
```suggestion
Merging a PR requires the following steps to be completed before it is merged automatically.
```

Reply:
I think in this case we should be extra clear and repeat the term.